Useful links/Useful links.md

```diff
@@ -4,37 +4,38 @@ Here are some useful links that will help you during the onboarding, but will al
 
 [WordPress Developer Reference](https://developer.wordpress.org/reference/)
 
-[WordPress Core Handbook](https://make.wordpress.org/core/handbook/)  
-[WordPress Themes Handbook](https://make.wordpress.org/themes/handbook/)  
-[WordPress Plugins Handbook](https://make.wordpress.org/plugins/handbook/)  
-[Gutenberg Documentation](https://developer.wordpress.org/block-editor/)  
+[WordPress Core Handbook](https://make.wordpress.org/core/handbook/)
```

Comment:
The two spaces at the end are there on purpose, this is how markdown knows to put things in the new line, please revert this change 🙂

Reply:
Oh sorry I will revert this